src/Controller/EventController/Handler/ShowEventHandler/ShowEventHandler.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Controller\EventController\Handler\ShowEventHandler;
+
+use App\Controller\EventController\Dto\ShowEventDto;
+use App\Entity\Event;
+use App\Entity\User;
+use App\Repository\EventRepository;
+
+class ShowEventHandler implements ShowEventHandlerInterface
+{
+    public function __construct(private readonly EventRepository $repository)
+    {
+    }
+
+    public function handle(ShowEventDto $dto, User $user): array{
+        $event = $this->repository->findOneBy(['uuid' => $dto->getUuid()]);
```

Comment:
Maybe better to create method in repository where you will get event by uuid?
(and if it's not exists will throw exception?)